lib/search_files.js

```diff
@@ -0,0 +1,291 @@
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
+var glob = require('glob');
+
+
+function isWindows() {
+  return process.platform === 'win32';
+}
+
+function isFunction(v) {
+  return (typeof v === 'function');
+}
+
+function isObject(v) {
```

Comment:
Load time of 'lodash/isObject' is as short as 'lodash.isobject' and other 'lodash/is*' modules are also same. But 'lodash/mapValues' takes nearly 50ms because it would load many modules via require tree. (lodash.mapvalues takes less than 10ms and it loads only three modules.)

And this isObject function corresponds to lodash.isplainobject.